cpp/terrain/terrain_object.h

```diff
@@ -81,9 +87,19 @@ class TerrainObject : public std::enable_shared_from_this<TerrainObject> {
 	Unit &unit;
 
 	/**
+	 * is the object a floating outline
+	 */
+	bool is_floating() const;
```

Comment:
when would an object be floating (more docs please, I have no idea what you're talking about here)